docs/src/loop.rst

```diff
@@ -71,9 +71,10 @@ API
 
 .. c:function:: int uv_loop_close(uv_loop_t* loop)
 
-    Closes all internal loop resources. This function must only be called once
-    the loop has finished its execution or it will return UV_EBUSY. After this
-    function returns the user shall free the memory allocated for the loop.
+    Releases all internal loop resources. Call this function only when the loop
+    has finished executing and all open handles and requests have been closed,
+    or it will return UV_EBUSY. After this function returns the user can free
```

Comment:
I'm no English expert, but it looks like in this case it could make sense. See section `Dependent Clause Before Independent Clause` @ http://theeditorsblog.net/2014/07/30/commas-with-subordinate-clauses-a-readers-question